src/components/inputs/LInputLoaded.vue

```diff
@@ -11,14 +11,15 @@
       :attach="!outlined"
       :solo="!outlined"
       :background-color="inputBackgroundColor"
+      :height="!allowHeightGrow ? inputHeight : ''"
```

Comment:
Aqui da pra manter o height normal mesmo
```suggestion
      :height="inputHeight"
```